File: src/test/java/edu/hm/hafner/analysis/IssuesSoftAssertions.java
package edu.hm.hafner.analysis;

import org.assertj.core.api.SoftAssertions;

/**
 * Custom soft assertions for {@link Issues}.
 *
 * @author Joscha Behrmann
 */
public class IssuesSoftAssertions extends SoftAssertions {

    public IssuesAssert assertThat(Issues actual) {
        return proxy(IssuesAssert.class, Issues.class, actual);
    }
}

File: src/test/java/edu/hm/hafner/analysis/IssuesSoftAssertions.kt
package edu.hm.hafner.analysis

import edu.hm.hafner.analysis.assertj.SoftAssertions

class IssuesSoftAssertions : SoftAssertions() {
    fun assertThat(actual: Issues): IssuesAssert {
        return proxy(IssuesAssert::class.java, Issues::class.java, actual)
    }
}
